UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/SceneImporter/ImporterMeshes.cs

```diff
@@ -34,9 +48,83 @@ protected virtual async Task ConstructMesh(GLTFMesh mesh, int meshIndex, Cancell
 				return;
 			}
 
+			var firstPrim = mesh.Primitives.Count > 0 ?  mesh.Primitives[0] : null;
+
+#if UNITY_DRACO
+			var anyHadDraco = false;
+			Mesh[] meshes = new Mesh[mesh.Primitives.Count];
+			var meshDataArray = Mesh.AllocateWritableMeshData(mesh.Primitives.Count);
+			var dracoDecodeResults = new DracoMeshLoader.DecodeResult[mesh.Primitives.Count];
+			for (int i = 0; i < mesh.Primitives.Count; i++)
+			{
+				var primitive = mesh.Primitives[i];
+				if (primitive.Extensions == null || !primitive.Extensions.ContainsKey("KHR_draco_mesh_compression"))
```

Comment:
Can we use the string constant instead of the verbatim string in the various places here?